src/services/requestService.js

```diff
@@ -39,8 +38,38 @@ export default class RequestService {
     return updatedRequest[1][0].dataValues;
   }
 
+  /**
+  * update trip request
+  * @param {number} id - request id
+  * @param {object} data - request object
+  * @return {object} - updated request
+  */
+  static async respondToRequest({ body: { status }, params: { requestId }, user: { id } }) {
+    const userRequest = await Request.findOne({ where: { id: requestId } });
```

Comment:
i think you also need to check if the request is still open here. Or is there a reason why you did not check for that ?

Reply:
That's because the PO wants the manager to be able to reject requests that have been previously approved as well

Reply:
Ok, that is true